### PR TITLE
Add NR_TAGS env var to allow users to specify tags in the format of k…

### PR DESCRIPTION
**Pull request by prodjito**
…ey1:value1;key2:value2 to be applied to all log events

### CODE REVIEW

File: src/function.py [hunk 1]
```diff
@@ -453,6 +461,11 @@ def _package_log_payload(data):
         }
     ]
 
+    nr_tags_str = _get_newrelic_tags()
+    if nr_tags_str:
```
Comment (kolanos) [inline src/function.py:465]: Why not put this logic in the `_get_newrelic_tags` function?

**Review by kolanos — CHANGES_REQUESTED** [review]
The linter check is failing. It is mostly due to all the extra spaces.

You can fix it with:

```bash
pip3 install black
black
```

File: src/function.py [hunk 2]
```diff
@@ -453,6 +461,11 @@ def _package_log_payload(data):
...
+        nr_tags = dict(item.split(":") for item in _get_newrelic_tags().split(";"))
```
Comment (MattWhelan) [inline src/function.py:466]: Let's also filter out collisions; the user should not overwrite the `plugin` or `aws` keys.

**Review by MattWhelan — CHANGES_REQUESTED** [review]

**Comment by MattWhelan** [comment]
I like the idea of this change; thanks for the contribution. If you could encapsulate the parsing logic in the getter, and fix that lint issue, then I'd be happy to merge this.

File: src/function.py (hunk 1 above)
Comment (prodjito) [reply src/function.py]: Thanks for the feedback. I have made this change.

File: src/function.py (hunk 2 above)
Comment (prodjito) [reply src/function.py]: I have made this change as well.

**Review by MattWhelan — APPROVED** [review]

**Review by MattWhelan — APPROVED** [review]

**Pull request merged**